registration-processor/pre-processor/registration-processor-quality-checker-stage/src/main/java/io/mosip/registration/processor/quality/checker/stage/QualityCheckerStage.java

```diff
@@ -152,10 +159,6 @@
 	@Autowired
 	private PriorityBasedPacketManagerService packetManagerService;
 
-	/** The cbeff util. */
-	@Autowired
```

Comment:
Do we need to remove this ?

Reply:
These beans are unused in QualityCheckerStage